examples/tv-casting-app/tv-casting-common/support/CastingStore.cpp

```diff
@@ -521,9 +522,9 @@ CHIP_ERROR CastingStore::WriteAll(std::vector<core::CastingPlayer> castingPlayer
             ReturnErrorOnFailure(tlvWriter.EndContainer(deviceTypeListContainerType));
 
             // ServerList container starts
-            chip::TLV::TLVType serverListContainerType = chip::TLV::kTLVType_Array;
+            chip::TLV::TLVType serverListContainerType;
             ReturnErrorOnFailure(tlvWriter.StartContainer(chip::TLV::ContextTag(kCastingPlayerEndpointServerListContainerTag),
-                                                          chip::TLV::kTLVType_Array, serverListContainerType));
```

Comment:
is this a backward incompatible change? Should we have a encoding version field to avoid errors that might be otherwise difficult to diagnose?

Reply:
It is backwards compatible because this change has been made in the CastingStore which is used by the newer APIs, as opposed to the PersistenceManager which is used by the older implementation. The CastingStore and PeristenceManager store the TLV data against different keys in the Key Value Store. And no client is using the CastingStore just yet. So anyone using the PersistenceManager implementation would be unaffected.